src/modules/callouts/pages/CalloutPage.vue

```diff
@@ -151,6 +177,15 @@ const canRespond = computed(
     currentUser.value?.activeRoles.includes('member')
 );
 
+// edge-case where a member is not contributing
+// (anymore) and should be given some indication as
+// to why they can't reply to a callout
+const canSeeButNotRespond = computed(
+  () =>
+    callout.value?.access === 'member' &&
```

Comment:
I think this might be missing checking if they are logged in (if not they see a login prompt a la `showLoginPrompt` below. So maybe this should check something like
```ts
callout.value?.access === 'member' && currentUser.value && !currentUser.value.activeRoles.includes('member')
```

Reply:
Thank you that's really well spotted